.travis.yml

```diff
@@ -15,8 +18,13 @@ jobs:
   include:
     - stage: lint
       script: make lint
-    - stage: ci
-      script: make ci
+    - stage: Lint
```

Comment:
Is this not a duplicate of the previous?

Reply:
Yes, it is, thanks, will fix.